### PR TITLE
design: update 2019-04-11-indexmerge.md with some edits

### PR DESCRIPTION
**Pull request by dcalvin**
### What problem does this PR solve?  

Some edits on the language @eurekaka  Please make sure  the edits don't alter any technical meaning as intended.

### CODE REVIEW

File: docs/design/2019-04-11-indexmerge.md [hunk 1]
```diff
@@ -6,94 +6,100 @@
...
+- `IndexMergeReader / PhysicalIndexMergeReader` 
+- `IndexMergeLookUpReader / PhysicalIndexMergeLookUpReader`.
+
+Now consider the following types of queries: 
 
-（1）Conditions in CNF, e.g, `select * from t1 where c1 and c2 and c3 and …`
+（1）Conditions in conjunctive normal form (CNF), e.g, `select * from t1 where c1 and c2 and c3 and …`
 
-In this form, each CNF item can be covered by a single index respectively. For example, if we have single column indexes for `t1.a`, `ta.b` and `t1.c` respectively, for SQL `select * from t1 where (a < 10 or a > 100) and b < 10 and c > 1000`, we can use all the three indexes to read the table handles. The result plan for it is like:
+In this form, each CNF item can be covered by a single index respectively. For example, if we have single column indexes for `t1.a`, `ta.b` and `t1.c` respectively, for SQL `select * from t1 where (a < 10 or a > 100) and b < 10 and c > 1000`, we can use all the three indexes to read the table handles. The result plan for it is:
```
Comment (zz-jason) [inline docs/design/2019-04-11-indexmerge.md]:
```suggestion
In this form, each CNF item can be covered by a single index respectively. For example, if we have single column indexes for `t1.a`, `t1.b` and `t1.c` respectively, for SQL `select * from t1 where (a < 10 or a > 100) and b < 10 and c > 1000`, we can use all the three indexes to read the table handles. The result plan for it is:
```

File: docs/design/2019-04-11-indexmerge.md [hunk 2]
```diff
@@ -6,94 +6,100 @@
...
-We load two million rows into `T200M` with one to two million sequence for all columns. We alter `$1-$4` to obtain equal selectivity on `C1` and `C2`, while the difference between `CNF-1` and `CNF-2` is `CNF-1` has no final results. The result can be seen in the following graph:
-
-<img alt="CNF 200" src="./imgs/cnf200m2.png" width="500pt"/>
-
+	TableScan
+```
+
+For the CNF items not covered by any index, we take them as table filters and convert them to selections on top of the scan node. For SQL `select * from t1 where (a < 10 or c >100) and b < 10`, only item `b < 10` can function as an index access condition, so we will use a single index lookup reader.
+
+To compare our demo implementation against the master branch, we set up a experiment for the CNF form. The schema and test SQL form we defined are as below:
```
Comment (zz-jason) [inline docs/design/2019-04-11-indexmerge.md]:
```suggestion
To compare our demo implementation against the master branch, we set up an experiment for the CNF form. The schema and test SQL form we defined are as below:
```

Comment (zz-jason) [inline docs/design/2019-04-11-indexmerge.md]:
```suggestion
To compare our demo implementation against the master branch, we set up an experiment for the CNF form. The schema and test SQL form we defined are as below:
```

File: docs/design/2019-04-11-indexmerge.md [hunk 3]
```diff
@@ -6,94 +6,100 @@
...
-We load two million rows into `T200` with one to two million sequence for all columns. We alter the value of `$1` and `$2` in test sql form to obtain the accurate selectivities. The result can be seen in the following graph:
-
-<img alt="DNF 200" src="./imgs/dnf200.png" width="500pt"/>
+We load two million rows into table `T200` with one to two million sequence for all columns. The value of `$1` and `$2` in the test SQL form is altered to obtain the accurate selectivities. The result can be seen in the following graph:
+
+<img alt="DNF 200" src="./imgs/dnf200.png" width="500pt"/>
 
 
-We design PhysicalIndexMergeLookUpReader structure as:
+The structure of PhysicalIndexMergeLookUpReader is designed as:
```
Comment (zz-jason) [inline docs/design/2019-04-11-indexmerge.md]:
```suggestion
The structure of `PhysicalIndexMergeLookUpReader` is designed as:
```

File: docs/design/2019-04-11-indexmerge.md [hunk 4]
```diff
@@ -6,94 +6,100 @@
...
-Table Schema:
-	CREATE TABLE T200(a int, b int, c int); 
-	CREATE INDEX T200a on T2OO(a);
-	CREATE INDEX T200b on T200(b);
-
-Test SQL Form:
-	SELECT * FROM T200 WHERE a < $1 OR b > $2;
+```
+
+To compare our demo implementation against the master branch, we set up a experiment for the DNF form. The schema and test SQL form we defined are as below:
```
Comment (dcalvin) [inline docs/design/2019-04-11-indexmerge.md]:
```suggestion
To compare our demo implementation against the master branch, we set up an experiment for the DNF form. The schema and test SQL form we defined are as below:
```

**Comment by dcalvin** [comment]
@zz-jason PTAL. Thanks.

**Pull request merged**